internal/service/fw/api_security_policies_resource_test.go

```diff
@@ -641,9 +606,11 @@ resource "bloxone_td_security_policy" "test_safe_search" {
 }
 
 func testAccSecurityPoliciesRules(name, rulesAction, rulesData string) string {
+	listName1 := acctest.RandomNameWithPrefix("named-list")
+	listName2 := acctest.RandomNameWithPrefix("named-list")
```

Comment:
The names should be parameterized. Having it here calls update on these supporting resources unnecessarily.

Reply:
Resolved

Reply:
The change you made only removes the variable names. They are still initialized within this function. 
They should be initialized outside the function and passed as parameters

Reply:
Added them to the UT that calls that particular function.